### DISCUSSION TITLE
Osu!lazer maps folder

### DISCUSSION THREAD

**Discussion by Realxspz** [ppy/osu#17126] [Q&A] [answered]
i had to delete osu from my Pc due to some bug. now all my maps are gone only exeption for osu!lazer all maps
 are there but i cant find the folder to export my maps from lazer to normal osu

**Comment by bdach** [comment, marked as answer]
there is no easy way to do this sorry. lazer uses a completely different file structure.

**Reply by Realxspz** [reply]
damnit but thanks for your reply
